assets/wizards/popups/views/campaigns/index.js

```diff
@@ -0,0 +1,319 @@
+	const [ modalType, setModalType ] = useState();
+	const [ campaignName, setCampaignName ] = useState();
+	const [ inFlight, setInFlight ] = useState( false );
+
+	const history = useHistory();
+
+	const submitModal = modalText => {
+		if ( MODAL_TYPE_NEW === modalType ) {
+			createTerm( modalText );
+		} else if ( MODAL_TYPE_RENAME === modalType ) {
+			renameCampaignGroup( campaignId, modalText );
+		} else if ( MODAL_TYPE_DUPLICATE === modalType ) {
+			duplicateCampaignGroup( campaignId, modalText );
+		}
+		setModalVisible( false );
+	};
+
+	const createTerm = term => {
+		setPopoverVisible( false );
+		setInFlight( true );
+		wizardApiFetch( {
+			path: '/wp/v2/newspack_popups_taxonomy',
+			method: 'POST',
+			quiet: true,
+			data: {
+				name: term,
+				slug: term,
+			},
+		} )
+			.then( () => {
+				setInFlight( false );
+				setCampaignName( '' );
+				refetch();
+			} )
+			.catch( () => {
+				setInFlight( false );
+			} );
+	};
+
+	const activeCampaigns = campaigns.filter( ( { status } ) => 'archive' !== status );
+	const archivedCampaigns = campaigns.filter( ( { status } ) => 'archive' === status );
+	const campaignData = dataForCampaignId( campaignId, campaigns );
+
+	const valueForCampaignId = id => {
+		if ( 'unassigned' === id ) {
+			return {
+				key: 'unassigned',
+				name: __( 'Unassigned Prompts', 'newspack' ),
+			};
+		}
+
+		const data = dataForCampaignId( id, campaigns );
+
+		if ( data ) {
+			return {
+				key: data.term_id,
+				name: data.name,
+			};
+		}
+		return {
+			key: 'active',
+			name: __( 'Active Prompts', 'newspack' ),
+		};
+	};
+	const selectValue = valueForCampaignId( campaignId );
+	return (
+		<Fragment>
+			<div className="newspack-campaigns__campaign-group__filter-group-wrapper">
+				<div className="newspack-campaigns__campaign-group__filter-group-actions">
+					<CustomSelectControl
+						label={ __( 'Campaigns', 'newspack' ) }
+						options={ [
+							{
+								key: 'active',
+								name: __( 'Active Prompts', 'newspack' ),
+								className: selectValue.key === 'active' && 'is-selected',
+							},
+							hasUnassigned && {
+								key: 'unassigned',
+								name: __( 'Unassigned Prompts', 'newspack' ),
+								className: selectValue.key === 'unassigned' && 'is-selected',
+							},
+							activeCampaigns.length && {
+								key: 'header-campaigns',
+								name: __( 'Campaigns', 'newspack' ),
+								className: 'is-header',
+							},
+							...activeCampaigns.map( ( { term_id: id, name } ) => ( {
+								key: id,
+								name,
+								className: classnames(
+									'newspack-campaigns__campaign-group__select-control-group-item',
+									+selectValue.key === +id && 'is-selected'
+								),
+							} ) ),
+							archivedCampaigns.length && {
+								key: 'header-archived-campaigns',
+								name: __( 'Archived Campaigns', 'newspack' ),
+								className: 'is-header',
+							},
+							...archivedCampaigns.map( ( { term_id: id, name } ) => ( {
```

Comment:
Wondering if we should show a label or style when viewing an archived campaign, so they're visually distinguishable from non-archived campaigns?

Reply:
Would this be in the Select itself, like this?

<img width="1539" alt="Screen Shot 2021-02-01 at 7 16 31 PM" src="https://user-images.githubusercontent.com/1477002/106534437-11343800-64c2-11eb-8ced-1de6e93387f6.png">

Reply:
That could work—I just found myself wanting to know at a glance somehow whether the campaign I'm looking at is archived.

Reply:
Done in https://github.com/Automattic/newspack-plugin/pull/833/commits/466d234ac68ec2098f19c591ee4e7abead3b14a1